tests/Benchmarks/Buffers/Writer.cs

```diff
@@ -11,9 +11,8 @@
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Text.Utf8;
-using System.Threading;
 
-public class BufferWriterBench
+public class Writer
```

Comment:
Let's set a consistent structure for the namespaces. I like the Benchmarks.Encodings for the other tests. For this one, can we put it in the Benchmarks.Buffers namespace?